compiler/optimizer/VPHandlers.cpp

```diff
@@ -447,6 +446,25 @@ static bool tryFoldCompileTimeLoad(
                }
             return didSomething;
             }
+         else if (node->getSymbolReference() == vp->comp()->getSymRefTab()->findVftSymbolRef()
+                  && node->getFirstChild() == curNode
+                  && constraint->isNonNullObject()
+                  && constraint->getClassType()
+                  && constraint->getClassType()->asFixedClass()
+                  && constraint->getClass()
+                  && !vp->comp()->compileRelocatableCode())
+            {
+            if (vp->trace())
+               traceMsg(vp->comp(), "VP Transforming VFTLoad to loadaddr: as n%dn is VFT load of known class", node->getGlobalIndex());
+            node->setNumChildren(0);
+            TR::Node::recreate(node, TR::loadaddr);
```

Comment:
Since you're setting the symref too, this might be a good place to use `recreateWithSymRef()`

Reply:
Changed the `recreate` with `recreateWithSymRef` in https://github.com/eclipse/omr/compare/c35d09156ee864be4bac1719ab8570f4d8a88d04..8066b3dfe38dd091f069482ff4d5daf4cefce3f8